demo/blocks_plug/run.py

```diff
@@ -22,13 +22,12 @@ def change_tab():
 
 with gr.Blocks() as demo:
     gr.Markdown("Three demos in one!")
-    with gr.Tabs(selected=1) as tabs:
-        with gr.TabItem("Text Identity", id=0):
-            identity_demo.render()
-        with gr.TabItem("Text Input", id=1):
-            input_demo.render()
-        with gr.TabItem("Text Static", id=2):
-            output_demo.render()
+    with gr.Tab("Text Identity", id=0):
```

Comment:
This breaks the demo since the `tabs` variable is no longer defined!

Reply:
Good catch!